.github/ISSUE_TEMPLATE/noisepage-issue-template.md

```diff
@@ -38,10 +38,14 @@ Please provide a short summary of the bug observed.
 ## Environment
 To address the bug, especially if it environment specific, we need to know what kind of configuration you are running on. Please include the following:
 
-**OS**: Ubuntu 18.04 (LTS) or macOS 10.14+ (specify version).
+**OS**: Ubuntu (LTS) 18.04-20.04 or macOS 10.14+ (please specify version).
 
 **Compiler**: GCC 7.0+ or Clang 8.0+. 
 
+**CMake Profile**: Debug/Release or maybe even FastDebug/Relwithdebinfo. If exists across all cmake profiles on a platform-compiler combo, then please say so.
```

Comment:
Spelling/wording: 

Debug, Release, FastDebug, or RelWithDebInfo. If the issue exists across all CMake profiles on an OS-compiler combo, please mention.

Reply:
+1, maybe also format as `Debug`, `Release`, ...

Reply:
done